runtime/criusupport/criusupport.cpp

```diff
@@ -706,6 +707,21 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			break;
 		}
 
+		if (NULL != vm->checkpointState.restoreArgsList) {
```

Comment:
can you move this to `runInternalJVMRestoreHooks`

Reply:
we'll need a way for `runInternalJVMRestoreHooks` to return an nls message

Reply:
Moved it into `runInternalJVMRestoreHooks()` which now returns NLS messages.